homeassistant/components/automation/mqtt.py

```diff
@@ -11,15 +11,20 @@
 # mypy: allow-untyped-defs
 
 CONF_ENCODING = "encoding"
+CONF_QOS = "qos"
```

Comment:
Why this change?

Reply:
Changed to allow setting QoS, it's supported by all MQTT platforms except automation.
(MQTT device triggers is using mqtt automation internally)

Reply:
I guess this PR wasn't big enough yet 😆